sdk/include/psdk/shellapi.h

```diff
@@ -638,6 +638,12 @@ DoEnvironmentSubstW(
     _Inout_updates_(cchSrc) LPWSTR pszSrc,
     UINT cchSrc);
 
+HRESULT WINAPI
+SHSetUnreadMailCountW(
```

Comment:
SHSetUnreadMailCountA is missing in this PR.

Reply:
There is no `SHSetUnreadMailCountA` for WinXP/Win2k3.
MS Learn is wrong.